tests/pushbits/main.c

```diff
@@ -77,8 +77,17 @@ echo_stdin_stdout(void)
 	/* Wait a short while, then cancel the thread. */
 	sleep(1);
 	if ((rc = pthread_cancel(thread)) != 0) {
-		warn0("pthread_cancel: %s", strerror(rc));
-		goto err0;
+		/*
+		 * These tests ignore an ESRCH error returned by
+		 * pthread_cancel.  According to the POSIX standard, a
+		 * thread ID should still be valid once it finishes if
```

Comment:
"once it finishes" is a bit imprecise (does the thread ID finish?) so I'd go with "after pthread_exit has been invoked by the thread".